src/pages/Dashboard/OverviewCard.tsx

```diff
@@ -418,3 +412,18 @@ export function OverviewCard({
     </div>
   );
 }
+
+function getFormattedDuration() {
+  const epochStartedTimestamp = getCurrentEpochStartedTimestamp();
+
+  const now = Date.now() / 1000;
+  const days = Math.floor((now - epochStartedTimestamp) / (3600 * 24));
+  let restHours = Math.round((now - epochStartedTimestamp) / 3600) - days * 24;
+  if (days === 0) {
+    restHours = Math.max(restHours, 1);
+  }
```

Comment:
Just a suggestion to consider usage `date-fns` for this date operations with :
```
import { differenceInDays, differenceInHours } from 'date-fns';


const now = new Date();
const epochStartedDate = new Date(epochStartedTimestamp * 1000);

const days = differenceInDays(now, epochStartedDate);
let restHours = differenceInHours(now, epochStartedDate) - days * 24;

if (days === 0) {
  restHours = Math.max(restHours, 1);
}
```
It's up to you which variant to keep, I'm also ok with yours